app/src/main/java/com/coveros/coverosmobileapp/blogpost/BlogPostsListActivity.java

```diff
@@ -35,6 +39,11 @@
     private SparseArray<String> authors = new SparseArray<>();  // to aggregate the ids and names of the authors of displayed blog posts
     private RequestQueue rQueue;
 
+    private static final String[] menuTitles = new String[]{"Website", "Blog", "Bookmarks"};
```

Comment:
A general style suggestion for constants is to name them using all caps and underscores (i.e. MENU_TITLES)